packages/react-menu/src/MenuMemoizedMenuItems.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Menu, MenuTrigger, MenuList, MenuItemCheckbox, MenuPopover, MenuItemCheckboxProps } from './index';
+
+import { Button } from '@fluentui/react-button';
+import { Edit20Regular as EditIcon } from '@fluentui/react-icons';
+
+const MemoCheckbox = React.memo((props: MenuItemCheckboxProps) => {
+  // use icons in the memo because JSX will always create a new object
+  // possible to memoize icons but it can be overkill
+  return (
+    <MenuItemCheckbox icon={<EditIcon />} name={props.name} value={props.value}>
+      {props.children}
+    </MenuItemCheckbox>
+  );
+});
+
+export const MemoizedMenuItems = () => {
```

Comment:
@PeterDraex - FYI this is yet another example of a story which shows a completely useless source code in storybook.